nerdlets/service-containers/heat-maps.js

```diff
@@ -110,7 +110,7 @@ export default class ContainerHeatMap extends React.PureComponent {
           return (
             <HeatMap
               title={title}
-              accountId={accountId}
+              accountIds={accountIds}
```

Comment:
This should be the only change needed in this file. The `HeatMap` props changed, but the this app hasn't changed, so we most likely don't want any changes besides this.
```suggestion
              accountIds={[accountIds]}
```